paper-tab.html

```diff
@@ -169,8 +164,25 @@
 
           anchor.click();
         }
-      }
+      },
+
+      _createRipple: function() {
+        var ripple = Polymer.PaperInkyFocusBehaviorImpl._createRipple();
+        ripple.removeAttribute('center');
+        ripple.classList.remove('circle');
+        return ripple;
+      },
 
+      _asyncClick: function() {
```

Comment:
Sure, but can you please document that? And most importantly, document how it's different, or why the original implementation doesn't work